content/en/real_user_monitoring/mobile_and_tv_monitoring/flutter/setup.md

```diff
@@ -243,6 +243,8 @@ The SDK changes its behavior according to the new value. For example, if the cur
 
 ## Automatically track views
 
+If you are using Flutter Navigator v2.0, your setup for automatic view tracking differs depending on your routing middleware. See [Flutter Integrated Libraries][7] for instructions on how to integrate with [go_router][8], [AutoRoute][9], and [Beamer][10].
```

Comment:
```suggestion
If you are using Flutter Navigator v2.0, your setup for automatic view tracking differs depending on your routing middleware. See [Flutter Integrated Libraries][12] for instructions on how to integrate with [go_router][7], [AutoRoute][9], and [Beamer][10].
```
These two are linking to the wrong place, based on your links below these should be the correct ones :)